src/zmq/zmqabstractnotifier.h

```diff
@@ -28,6 +30,12 @@ class CZMQAbstractNotifier
     void SetType(const std::string &t) { type = t; }
     std::string GetAddress() const { return address; }
     void SetAddress(const std::string &a) { address = a; }
+    int GetOutboundMessageHighWaterMark() const { return outbound_message_high_water_mark; }
+    void SetOutboundMessageHighWaterMark(const int sndhwm) {
+        if ((sndhwm >= 0) && (sndhwm <= std::numeric_limits<int>::max())) {
```

Comment:
```
2018-09-25 21:25:16 clang(pr=14060): zmq/zmqabstractnotifier.h:35:38: warning: result of comparison 'const int' <= 2147483647 is always true [-Wtautological-type-limit-compare]
```

Reply:
ok, i'll just check it like zmq checks it: https://github.com/zeromq/libzmq/blob/master/src/options.cpp#L310-L315